Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,4381 @@
+    for arg, value in args.items():
+        if arg in ['issueId', 'issueIdOrKey']:
+            # In v2, there was no differentiation between issue id and key arguments,
+            # and in v3, we do differentiate.
+            if is_issue_id(value):
+                v3_args['issue_id'] = value
+            else:
+                v3_args['issue_key'] = value
+        elif arg in v2_args_to_v3:
+            v3_args[v2_args_to_v3[arg]] = value
+        else:
+            v3_args[arg] = value
+    return v3_args
+
+
+def main() -> None:
+    params: Dict[str, Any] = demisto.params()
+    # args: Dict[str, Any] = demisto.args()
+    args = map_v2_args_to_v3(demisto.args())
+    verify_certificate: bool = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    # Cloud configuration params
+    cloud_id = params.get('cloud_id', '')
+    client_id = params.get('client_id', '')
+    client_secret = params.get('client_secret', '')
+    callback_url = params.get('callback_url', '')
+
+    # OnPrem configuration params
+    server_url = params.get('server_url', 'https://api.atlassian.com/ex/jira')
+
+    # Fetch params
+    issue_field_to_fetch_from = params.get('issue_field_to_fetch_from', 'id')
+    fetch_query = params.get('fetch_query', 'status!=done')
+    id_offset = params.get('id_offset', '0')
+    fetch_attachments = argToBoolean(params.get('fetch_attachments', False))
+    fetch_comments = argToBoolean(params.get('fetch_comments', False))
+    max_fetch = params.get('max_fetch', DEFAULT_FETCH_LIMIT)
+    fetch_interval = params.get('incidentFetchInterval', DEFAULT_FETCH_INTERVAL)  # in minutes
+    # This is used in the first fetch of an instance, when issue_field_to_fetch_from is either, update date, or created date,
+    # it holds values such as: 3 days, 1 minute, 5 hours,...
+    first_fetch_interval = params.get('first_fetch', DEFAULT_FIRST_FETCH_INTERVAL)
+    mirror_direction = params.get('mirror_direction', 'None')
+    comment_tag_to_jira = params.get('comment_tag_to_jira', 'comment tag')
+    comment_tag_from_jira = params.get('comment_tag_from_jira', 'comment tag from Jira')
+    if comment_tag_to_jira == comment_tag_from_jira:
+        raise DemistoException(('Comment Entry Tag to Jira and Comment Entry Tag '
+                               'from jira cannot have the same value.'))
+
+    attachment_tag_to_jira = params.get('attachment_tag_to_jira', 'attachment tag')
+    attachment_tag_from_jira = params.get('attachment_tag_from_jira', 'attachment tag from Jira')
+    if attachment_tag_to_jira == attachment_tag_from_jira:
+        raise DemistoException(('Attachment Entry Tag to Jira and Attachment Entry Tag '
+                               'from jira cannot have the same value.'))
+    # Mirroring params
+    mirror_resolved_issue = argToBoolean(params.get('close_incident', False))
+    # Print to demisto.info which Jira instance the user supplied.
+    # From param or if automatically
```

Comment:
Is this a TODO comment?

Reply:
Yes, but I decided to print it to **demisto.debug**, and not to **demisto.info**, since I want to use this information whenever we need to deal with a bug of some type.